### ISSUE TITLE
How to make new record auto take FK in data fill-without user manual select

### ISSUE THREAD

**Issue by itachiemashita89** [saltcorn/saltcorn#1766]
Hi,
I do have 2 tables name "branch" & "project" which "project" got Key to branch as FK. In adding row from the table I do able see the record to "branch" which shows the FK is relation is functioning well.
My question is, in the add form UI, is there any way to get this Join Field auto-detect/add as a record related to the FK and PK?
I mean the function is similar to the fixed field "present Staff ID" = Loggedin can auto-add the user ID in a new record.

![Screenshot 2023-04-10 123935](https://user-images.githubusercontent.com/91925990/230827206-b3d83a24-4ce0-4752-86e1-cdbccc421f77.png)

 As shown here, the branch from UI add record unable get any FK.  * 2 records above edit direct from the table.
I used "Joint Field" but could not get any record add.
Sorry if my explanation is confusing.
Thanks.

**Comment by gianlucafiore** [comment]
Hola! Buenos días, cuando creas una vista de tipo editar, y no incluyes el campo de relación, cuando avances te va a preguntar qué quieres hacer con ese campo, si lo dejas en blanco va a ser autocompletado con id FK padre desde donde invocas la vista.

Ejemplo: si tenés una vista de persona de tipo "show", con una list view anidada de mascotas, cuando le incluyas un botón add, para crear una nueva mascota, ésta se generará con el FK de la persona automáticamente

**Comment by SebaBecks** [comment]
English please

**Comment by gianlucafiore** [comment]
> English please

I had to learn to read English and I take the nice trouble to help...
but hey, as simple as using google translator:

Hello! Good morning, when you create an edit-type view, and you don't include the relationship field, when you advance it will ask you what you want to do with that field, if you leave it blank it will be auto-completed with the parent FK id from where you invoke the view .

Example: if you have a person view of the "show" type, with a nested pet list view, when you include an add button to create a new pet, it will be automatically generated with the person's FK

**Comment by itachiemashita89** [comment]
Hello, good morning @gianlucafiore 

Thank you for your respond.
![Screenshot 2023-04-11 103729](https://user-images.githubusercontent.com/91925990/231041310-76c7a970-a784-4289-bdfe-381cbf298149.png)
You mean something like this right? Just use this function?
I did do, but not for what column I want for. Others fill (FK) able pickup the fill item as I want.
I hope something wrong with my table relationship.
Will find out from your idea given.
Thanks a lots

**Comment by gianlucafiore** [comment]
Sí, el problema creo que viene en cómo está relacionada la tabla branches, o cómo está creada dicha tabla, si podés propiciar nos más info, saludos!
